testing/directives-test.js

```diff
@@ -19,19 +18,25 @@
             rootScope = $rootScope;
         }));
 
-        */
-/*it('Check the directive works', function(){
-            var directiveElementHtml = '<game-square number="0"></game-square>';
+it('Check the square selection directive works', function(){
+            var directiveElementHtml = '<game-square squareNumber="0"/>';
             var element = compile (directiveElementHtml)(rootScope);
             rootScope.$digest();
 
-            expect(element[0].toString()).to.be.equal('object HTMLElement');
-            expect(element.attr(number)).to.be.equal('0');
-            expect(element[0].children.length).to.equal(1);
-        });*//*
+            expect(element[0].toString()).to.be.equal('[object HTMLElement]');
+            expect(element.attr('squareNumber')).to.be.equal('0');
```

Comment:
This only tests on attribute - and not the important one. The ones missing are:
src
class
ng-click